Week2/homework/Masoud/js-exercises/EX1-removeTheComma.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+console.log(myString.length);
+
+//----My way--> I like it
+let changeStringToArray = [];
+for (let i = 0 ; i < myString.length; i++){
+  if (myString[i] == ','){
+    changeStringToArray[i] =  ' ';
```

Comment:
This will break if the first character of myString is a comma. but not a bad solution if you have to implement it yourself (once that bug is fixed)